cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
+\end{example}
+
+Nell'esempio sopra vediamo che \emph{gli oggetti non sono insiemi con strutture ulteriori}. 
+Notiamo anche che la proprietà di \emph{simmetria} delle relazioni di equivalenza non serve per avere una categoria. Consideriamo i seguenti esempi.
+
+\begin{examples}
+ Una relazione d'ordine $(X,\le)$ si può vedere come una categoria, ancora una volta con un unico morfismo $x\to y$ se e solo se $x\le y$. Come nel caso delle relazioni di equivalenza, le identità sono date dalla proprietà riflessiva e la composizione dalla proprietà transitiva. 
+ 
+ Più in generale, un \emph{preordine} è una relazione riflessiva e transitiva, ma non necessariamente simmetrica o antisimmetrica. Ogni preordine si può vedere come una categoria.
+\end{examples}
+
+
+\begin{example}
+ Un monoide si può vedere come una categoria con un solo oggetto -- più precisamente, una categoria \emph{localmente piccola} con un solo oggetto. 
+ Vediamo come. Dato un monoide $M$, definiamo la seguente categoria, che indichiamo con $\ctB M$.\footnote{A volte la categoria definita in questo modo si indica con $\ctB M$, a volte semplicemente con $M$.}
+ \begin{itemize}
+  \item La categoria $\ctB M$ ha un unico oggetto, che indichiamo con un punto, $\bullet$;
+  \item La categoria $\ctB M$ ha un morfismo $m:\bullet\to\bullet$ per ogni elemento $m\in M$;
+  \item L'identità dell'unico oggetto $m$ e il morfismo definito dall'elemento neutro $e\in M$;
+  \item La composizione di morfismi è data dal prodotto in $M$.
+ \end{itemize}
+ In un monoide, a differenza di una categoria generica, possiamo sempre comporre due morfismi $m$ e $n$: questo è garantito dal fatto che siccome c'è un solo oggetto, il dominio e il codominio di $m$ ed $n$ sono necessariamente uguali. 
+ 
+ Viceversa, data una categoria localmente piccola con un solo oggetto (indichiamolo ancora con questo simbolo, $\bullet$), l'insieme di morfismi $\bullet\to\bullet$ ha una struttura di monoide con l'elemento neutro dato dall'identità, e il prodotto dato dalla composizione.
+ 
+ L'unitalità e l'associatività del monoide e della categoria si corrispondono.
+\end{example}
+
+In particolare, ogni gruppo si può vedere come una categoria con un solo oggetto. 
+
+\begin{remark}
+ Abbiamo visto che i gruppi sono gli oggetti della categoria $\ctGrp$, ma anche che ogni gruppo si può a sua volta vedere come una categoria (con un solo oggetto). Entrambe le prospettive sono valide, e ce ne sono altre ancora. 
+ Allo stesso modo, gli ordini parziali sono categorie, ma si possono anche vedere come oggetti della categoria $\ctPos$, dove i morfismi sono le funzioni monotone. 
+ Spesso, in teoria delle categorie, la stessa struttura matematica può apparire in modi diversi in contesti diversi, e questa varietà di prospettive è uno dei motivi per cui le categorie sono così versatili. 
+\end{remark}
+
+
+Tutte le categorie descritte finora sono localmente piccole. 
+
```

Comment:
...Ne chiameremo mai in causa una che non lo è?

Reply:
Le categorie di funtori non saranno localmente piccole, giusto?

Reply:
(E ne parlerò, almeno un pochino, quando scrivo della categoria dei funtori.)